quinn/dataframe_helpers.py

```diff
@@ -86,12 +87,17 @@ def print_athena_create_table(
     s3location: str,
 ) -> None:
     """Generate the Athena create table statement for a given DataFrame.
-
     :param df: The pyspark.sql.DataFrame to use
     :param athena_table_name: The name of the athena table to generate
     :param s3location: The S3 location of the parquet data
     :return: None
     """
+    warnings.warn(
+        "Function may be removed in future versions of quinn. Please use explicit functions instead",
```

Comment:
I would suggest adding it here that the function will be removed in veraion 1.0. Because we already discussed this topic in Slack.

Reply:
> Function may be removed in version 1.0 of quinn. Please use explicit functions instead

Is this message fine? @SemyonSinchenko

Reply:
Also, there is no explicit function for it. So, just a warning that it will be removed in 1.0

Reply:
`print_athena_create_table` is a function, right?

Reply:
Exactly.